examples/example_evo_regression.py

```diff
@@ -1,122 +1,183 @@
 import numpy as np
 import matplotlib.pyplot as plt
+import os
 import scipy.constants
 import sympy
-from sympy.printing.dot import dotprint
+from functools import partial
```

Comment:
i would `import functools` to not loose the namespace on `partial`, but maybe it's common enough that it's clear where `partial` comes from